evap/evaluation/migrations/0148_evaluation_main_language.py

```diff
@@ -0,0 +1,29 @@
+# Generated by Django 5.1.3 on 2024-11-25 20:19
+
+from django.db import migrations, models
+
+
+def _migrate(apps, _schema_editor):
+    _approved_status = 40
+    Evaluation = apps.get_model("evaluation", "Evaluation")
+    Evaluation.objects.filter(state__gte=_approved_status).update(main_language="de")
```

Comment:
Should the default really be German here? @janno42 ?

Reply:
Please change it to English